README.md

```diff
@@ -61,3 +61,15 @@ A GitHub authentication. Used for getting details of a Pull Request from the Git
 - Required: `true`
 
 A GitHub repository in the format *org/repo* e.g. *nhsevidence/NICE.TopHat*
```

Comment:
Only thing we're missing is an option in the readme for `packageRelativePath` but apart from that, looks good